internal/ibazel/ibazel.go

```diff
@@ -603,3 +603,13 @@ func keys(m map[string]struct{}) []string {
 	}
 	return keys
 }
+
+func findInLocalRepository(localRepositories map[string]string, repo string) (string, bool) {
```

Comment:
This is very interesting. I've not kept informed about what the current state of Bazel is on overrides. Two things I would like to see before I will merge this. 

1. Please add some comments that link to documentation of this behavior in the override context
2. I would also like to see some testing of this functionality to ensure it behaves in the right way

Reply:
There are [a whole bunch](https://github.com/bazelbuild/bazel-watcher/tree/master/internal/e2e) of e2e tests that you could model your test off of as well

Reply:
Looking at [The Bazel Query Reference](https://bazel.build/query/language) I'm not sure this gives me confidence on the translation scheme from `+` to actual filenames. Sorry, this is just a part of Bazel I have no familiarity with. Is there anything that might provide more context?